modules/flysystem/intro.md

```diff
@@ -9,19 +9,21 @@ Flysystem is made available in Drupal 8 by the [Flysystem module](https://www.dr
 * Dropbox
 * Rackspace
 * Amazon S3
+
 It's relatively easy to implement new adapters, and more are becoming available all the time.
 
-When joined with a flysystem adapter module, the flysystem module offers four things:
-* a syntax to define the access details for the remote storage location in Drupal's settings.php.
-* a storage setting for Drupal's core file field.
-* access to the Flysystem API so you can execute standard file system operations (read, write, list, etc.) on your remote storage from your custom code.
-* a synchronisation UI that enables you to manually synchronise local and remote storage locations.
+When joined with a Flysystem adapter module, the Flysystem module offers four things:
```

Comment:
Five things now :)

Maybe we should change to to "the following" things or something similar that won't require another change every time when we update the list.